src/SQLite.cs

```diff
@@ -2595,7 +2595,7 @@ public static bool IsMarkedNotNull (MemberInfo p)
 
 	public class SQLitePreparedStatement : IDisposable {
 		Sqlite3Statement Statement;
-		SQLiteConnection Connection { get;  set; }
+		SQLiteConnection Connection { get; set; }
```

Comment:
private setter?

Reply:
isn't the whole thing private by default?

Reply:
... yes. didn't notice that.